Configuration/TypoScript/constants.txt

```diff
@@ -0,0 +1,108 @@
+	catalog.count.pid =
+	# cat=ai130page/links/140; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-session-pinned.target
+	catalog.pinned.pid =
+	# cat=ai130page/links/150; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:catalog-suggest.target
+	catalog.suggest.pid =
+	# cat=ai130page/links/160; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:basket-standard.target
+	basket.pid =
+	# cat=ai130page/links/170; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:checkout-standard.target
+	checkout.pid =
+	# cat=ai130page/links/171; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:checkout-standard-summary-option-terms.target
+	checkout.terms.pid =
+	# cat=ai130page/links/172; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:checkout-standard-summary-option-terms-privacy.target
+	checkout.terms.privacy.pid =
+	# cat=ai130page/links/180; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:checkout-confirm.target
+	checkout.confirm.pid =
+	# cat=ai130page/links/190; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:checkout-update.target
+	checkout.update.pid =
+	# cat=ai130page/links/200; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:account-favorite.target
+	account.favorite.pid =
+	# cat=ai130page/links/210; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:account-watch.target
+	account.watch.pid =
+	# cat=ai130page/links/220; type=string; label=LLL:EXT:aimeos/Resources/Private/Language/extension.xlf:account-download.target
+	account.download.pid =
+
+	view.pluginNamespace = ai
+
+	# cat=ai140advanced/file/100; type=string; label=Aimeos common: Path to common base theme CSS shared between website and e-mails
+	theme.css.aimeos-common = EXT:aimeos/Resources/Public/Themes/elegance/common.css
+	# cat=ai140advanced/file/101; type=string; label=Aimeos: Path to base theme CSS
+	theme.css.aimeos = EXT:aimeos/Resources/Public/Themes/elegance/aimeos.css
+	# cat=ai140advanced/file/110; type=string; label=Aimeos: Aimeos related java script code
+	theme.js.aimeos = EXT:aimeos/Resources/Public/Themes/aimeos.js
+	# cat=ai140advanced/file/120; type=string; label=Aimeos detail: Aimeos detail related java script code
+	theme.js.aimeos-detail = EXT:aimeos/Resources/Public/Themes/aimeos-detail.js
+	# cat=ai140advanced/file/130; type=string; label=JQuery source: jQuery source
+	lib.js.jquery = https://code.jquery.com/jquery-3.1.1.min.js
+	# cat=ai140advanced/file/131; type=boolean; label=JQuery source external: Set if the source is an external link
+	lib.js.jquery.external = 1
+	# cat=ai140advanced/file/140; type=string; label=JQuery UI custom source: jQuery UI custom source
+	lib.js.jquery-ui = EXT:aimeos/Resources/Public/Themes/jquery-ui.custom.min.js
```

Comment:
Things are getting difficult with two very different versions because they can't be used at the same page. The only solution is to let the integrator set the final jQuery version on conflicts. I think, Aimeos can at least use jQuery 2.x as well because we avoid to use new features if not absolutely necessary.